core/src/main/java/io/confluent/connect/storage/StorageSinkConnectorConfig.java

```diff
@@ -93,6 +94,8 @@
   public static final String SCHEMA_CACHE_SIZE_DISPLAY = "Schema Cache Size";
 
   protected static final ConfigDef CONFIG_DEF = new ConfigDef();
+  public static final GenericClassRecommender FORMAT_CLASS_RECOMMENDER =
+      new GenericClassRecommender();
```

Comment:
Again, would `GenericClassRecommender<Format>`be a bit safer?

Reply:
I agree. But see above restriction re: modules